aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,536 @@
+.. _web-api-help-pages-using-swagger:
+
+ASP.NET Web API Help Pages using Swagger
+========================================
+
+By `Shayne Boyer`_
+
+Understanding the various methods of an API can be a challenge for a developer when building a consuming application. 
+
+Generating good documentation and help pages as a part of your Web API using `Swagger <a href="http://swagger.io/">`_ with the .NET Core implementation `Swashbuckle <a href="https://github.com/domaindrivendev/Ahoy">`_ is as easy as adding a couple of nuget packages and modifying the Startup.cs.
+
+This tutorial builds on the sample on :doc:`first-web-api`. 
+
+.. contents:: Sections:
+  :local:
+  :depth: 2
+
+Getting Started
+---------------
+There are 2 core components to Swashbuckle 
+
+- Swashbuckle.SwaggerGen - provides the functionality to scaffold your Web API and generate JSON Swagger documents that describe the objects, methods, return types, etc. 
+- Swashbuckle.SwaggerUI - an embedded version of the Swagger UI tool which uses the above documents for a rich customizable experience for describing the Web API functionality. This also includes built in test harness capabilities for the public methods.
+
+NuGet Packages
+--------------
+
+Getting the core components added to the project via nuget can be done using either the PowerShell command, modifying the package.json file directly, adding the package to the dependencies node, or by right clicking the references folder and selecting "Manage NuGet Packages" and searching for "Swashbuckle". Be sure to select the "Include Pre-Release" to get the latest version while in beta.
```

Comment:
That's a long list in a sentence. How about something like this:


You can add ``Swashbuckle`` with any of the following approaches:

- From the **Package Manager Console**: 

.. code-block:: bash

  Install=Package Swashbuckle -Pre

- Add Swashbuckle to *project.json*:

.. code-block:: none

  "Swashbuckle": "6.0.0-<latest beta>"
- In Visual Studio:
  
  - Right click your  project in Solution Explorer > **Manage NuGet Packages**
  - Enter Swashbuckle in the search box
  - Check **Include prerelease**
  - Set the **Package source** to nuget.org 
  - Tap the Swashbuckle package and then tap **Install**

I would remove the images, we use them mostly in intro tutorials and this is not an intro. 

I'd remove the project.json file, we only want to show what they need to add. That hard coded version will change frequently